cpp/src/arrow/memory_pool.h

```diff
@@ -160,6 +160,16 @@ ARROW_EXPORT MemoryPool* system_memory_pool();
 /// May return NotImplemented if jemalloc is not available.
 ARROW_EXPORT Status jemalloc_memory_pool(MemoryPool** out);
 
+/// \brief Set jemalloc memory page purging behavior for future-created arenas
+/// to the indicated number of milliseconds. See dirty_decay_ms and
+/// muzzy_decay_ms options in jemalloc for a description of what these do. The
+/// default is configured to 0 which releases memory more aggressively to the
```

Comment:
Update comment? It doesn't default to 0 anymore.

Reply:
done